src/server/test-suites/transformers/test-suite-status.js

```diff
@@ -74,9 +79,55 @@ function testSuiteStatus(service) {
         }
       }
     },
+    cdpSquidProxy: {
+      name: 'Proxy',
+      part: 2,
+      url: {
+        text: `${githubOrg}/cdp-squid-proxy`,
+        href: `https://github.com/${githubOrg}/cdp-squid-proxy`
+      },
+      status: {
+        text: cdpSquidProxy?.status
+          ? startCase(cdpSquidProxy?.status)
+          : unknownValue,
+        classes: statusTagClassMap(cdpSquidProxy?.status)
+      },
+      info: () => {
+        switch (cdpSquidProxy?.status) {
+          case creationStatuses.raised:
+          case creationStatuses.requested:
+          case creationStatuses.inProgress:
+            return `Setting up proxy access.`
+          case creationStatuses.created:
+          case creationStatuses.success:
+            return `Your test suite can make outbound calls via the proxy.`
```

Comment:
doesn't need to be a template literal, no variable so single quotes are fine

Reply:
Also can I suggest:

`Your new service can now make outbound calls via the proxy.`